src/Serilog.AspNetCore/SerilogApplicationBuilderExtensions.cs

```diff
@@ -26,6 +30,9 @@ public static class SerilogApplicationBuilderExtensions
         const string DefaultRequestCompletionMessageTemplate =
             "HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms";
 
+        static Func<HttpContext, LogEventLevel> DefaultGetLogLevel = 
+            ctx => ctx.Response.StatusCode > 499 ? LogEventLevel.Error : LogEventLevel.Information;
```

Comment:
`DefaultGetLevel` perhaps? Was the conversion to a method not pushed with the last changes?